src/rp2_common/boot_bootrom_headers/include/boot/bootrom_constants.h

```diff
@@ -255,6 +255,12 @@ typedef int (*bootrom_api_callback_generic_t)(uint32_t r0, uint32_t r1, uint32_t
 #define BOOTROM_NS_API_get_b_partition 7
 #define BOOTROM_NS_API_COUNT 8
 
+#define OTP_CMD_ROW_BITS                    0x0000ffffu
+#define OTP_CMD_ROW_LSB                     _u(0)
+#define OTP_CMD_WRITE_BITS                  0x00010000u
+#define OTP_CMD_WRITE_LSB                   _u(6)
+#define OTP_CMD_ECC_BITS                    0x00020000u
```

Comment:
For completeness, should there also be `OTP_CMD_ECC_LSB` ? :shrug:

Reply:
done